lib/libutee/tee_api_operations.c

```diff
@@ -1418,11 +1426,6 @@ TEE_Result TEE_AEEncryptFinal(TEE_OperationHandle operation,
 		goto out;
 	}
 
-	if (!srcData && !srcLen) {
```

Comment:
Jens, I think this code in func TEE_CipherDoFinal should also remove, what do you think?